src/app/pages/plugins/plugins-installed/plugins-installed.component.ts

```diff
@@ -249,6 +289,22 @@ export class PluginsInstalledListComponent {
               });
         }
       },
+      {
+        id: "upgrade",
+        label: T("Upgrade"),
+        onClick: (row) => {
+          const option = {
+            'plugin': true,
+          }
+          const dialogRef = this.dialog.open(EntityJobComponent, { data: { "title": T("Upgrading Plguin") }, disableClose: true });
+          dialogRef.componentInstance.setCall('jail.upgrade', [row[1], option]);
+          dialogRef.componentInstance.submit();
+          dialogRef.componentInstance.success.subscribe((res) => {
+            dialogRef.close(true);
+            this.snackBar.open(T("Successfully Upgraded plugin."), T('Close'), { duration: 5000 });
```

Comment:
This should be normal sentence case:
`` T('Successfully upgraded plugin.'), T('Close'), { duration: 5000 }); ``